configs/ci/roles/config-manager.json

```diff
@@ -8,6 +8,9 @@
       "platform_default": false,
       "version": 3,
```

Comment:
You'll need to bump the `"version"`  to `4` in all environment files for the seed changes to be picked up.